zegami_sdk/collection.py

```diff
@@ -344,6 +337,20 @@ def download_single(index, url):
         for i in range(len(images)):
             ordered.append(images[i])
         return ordered
+    
+    def delete_collection_images(self):
```

Comment:
This should be tweaked to a signature of: `delete_images_with_tag(self, tag='delete')`